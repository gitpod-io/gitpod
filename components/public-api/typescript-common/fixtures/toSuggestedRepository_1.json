{
  "url": "https://github.com/gitpod-io/gitpod",
  "projectId": "123",
  "projectName": "Gitpod",
  "repositoryName": "gitpod"
}
