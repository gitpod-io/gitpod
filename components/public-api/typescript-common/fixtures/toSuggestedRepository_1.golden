{
    "result": {
        "url": "https://github.com/gitpod-io/gitpod",
        "repoName": "gitpod",
        "configurationId": "123",
        "configurationName": "Gitpod"
    },
    "err": ""
}
